web/public/locales/de/routes.guildsettings.api.json

```diff
@@ -1,28 +1,28 @@
 {
   "heading": "API",
-  "explanation": "When enabled, this exposes a public API endpoint which returns general information about this guild via the following endpoint.",
-  "save": "Save settings",
-  "reset": "Reset token",
+  "explanation": "Wenn aktiviert, gibt shinpuru einen API-Endpunkt frei, der generelle Informationen über die Guild ausgibt.",
+  "save": "Einstellungen speichern",
+  "reset": "Token zurücksetzen",
   "settings": {
-    "heading": "Settings",
-    "enable": "Enable Guild API",
+    "heading": "Einstellungen",
+    "enable": "Guild API aktivieren",
     "allowedorigins": {
-      "heading": "Allowed Origins",
-      "description": "A comma separated list of origin URLs set via the <1>Access-Control-Allow-Origin</1> header in the API response. That means, that the information provided via this API endpoint are only requestable in a browser on a website which has one of the given URLs. If nothing is set, <2>*</2> is used as value for the header which allows every website to request the resource."
+      "heading": "Zugelassene Origins",
+      "description": "Eine Kommaseparierte Liste von Herkunfts-URLs, die in der API-Antwort über den <1>Access-Control-Allow-Origin</1>-Header gesetzt werden. Das bedeutet, dass die Informationen, die von dieser API angefragt werden, nur in einem Browser von einer Website abgerufen kann, die eine dieser URLs hat. Ist hier nichts gesetzt, wird <2>*</2> als Wert für den Header genutzt, was jeder Website erlaubt, auf diese Ressource zuzugreifen."
     },
     "token": {
-      "heading": "Access Token",
-      "description": "Specify a token to protect the endpoint against unauthorized access. When set, you need to pass the token either as a value of the <1>Authorization</1> header with the token type <2>bearer</2> or via the <3>token</3> URL parameter on reqeusting the resource.",
-      "unprotected": "The endpoint is currently unprotected!",
-      "protected": "The endpoint is protected.",
+      "heading": "Zugriffstoken",
+      "description": "Gib ein Token an, um den Endpunkt vor unautorisiertem Zugriff zu schützen. Wenn gesetzt, muss dieses Token bei einem API-Zugriff entweder über den <1>Authorization</1>-Header mit dem Token-Typ <2>bearer</2> oder über den URL-Parameter <3>token</3> angegeben werden.",
+      "unprotected": "Dieser Endpunkt ist momentan ungeschützt!",
+      "protected": "Dieser Endpunkt ist geschützt.",
```

Comment:
I don't think that this should have the added temporal side of "momentan", since this *should* be a permanent state and therefore not include anythinkg that might make a user think that this is something temporary. If you know what I mean